### DISCUSSION TITLE
r is undefined [Apr 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1889] [Discord Threads]
> 🙋‍♂️ **alexrtou**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### able to reproduce. we'll take a look.  
#  
> 👨‍🚀 **wingkwong**  
#### an issue is created here https://github.com/nocodb/nocodb/issues/1737  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/965352825867632742)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
